sw/host/opentitanlib/src/app/conf.rs

```diff
@@ -130,6 +128,21 @@ pub struct FlashConfiguration {
     pub driver: FlashDriver,
 }
 
+/// Defines the way to bootstrap the OpenTitan chip.
+#[derive(Deserialize, Clone, Debug)]
+pub struct BootstrapConfiguration {
```

Comment:
Is this used anywhere?

Reply:
No, it is not.  Not yet at least.  My plan is that the bootstrap command, rather than "knowing" about pins called "RESET" and "BOOTSTRAP", would apply a named bootstrap configuration.  I never got to modifying the bootstrap command to recognizing this struct.

I think there could be more to discuss, and I do not want to hold up the support for pullup and open-drain in the GPIO trait for this, so I have taken out this piece.